aiohttp/web_response.py

```diff
@@ -380,10 +380,11 @@ async def _prepare_headers(self) -> None:
             keep_alive = request.keep_alive
         self._keep_alive = keep_alive
 
-        version = request.version
+        version = request._version
```

Comment:
This one isn't safe because they are allowed to override `request` with their own object
```suggestion
        version = request.version
```